### DISCUSSION TITLE
Linking issue with libyaml

### DISCUSSION THREAD

**Discussion by yongluoFlex** [emqx/neuron#483] [General]
[按照指导文档装完依赖库，但是编译报]
![image](https://user-images.githubusercontent.com/67996128/167797339-13f558bf-7aa1-4358-9fb7-6bce15e1b3f8.png)

**Comment by gc87** [comment]
The libyaml dependency will be removed, as a temporary solution you can refer to #476 and #477 .

**Reply by gc87** [reply]
```bash
$ git clone -b 0.2.5 https://github.com/yaml/libyaml.git
$ cd libyaml && mkdir build && cd build 
$ cmake -DBUILD_SHARED_LIBS=OFF -DCMAKE_POSITION_INDEPENDENT_CODE=ON .. && make
$ sudo make install
```
